tests/baselines/reference/user/uglify-js.log

```diff
@@ -73,6 +82,7 @@ node_modules/uglify-js/lib/compress.js(5254,17): error TS2403: Subsequent variab
 node_modules/uglify-js/lib/compress.js(5347,37): error TS2554: Expected 0 arguments, but got 1.
 node_modules/uglify-js/lib/compress.js(5603,57): error TS2345: Argument of type 'any[]' is not assignable to parameter of type '[string | RegExp, (string | undefined)?]'.
   Property '0' is missing in type 'any[]' but required in type '[string | RegExp, (string | undefined)?]'.
+node_modules/uglify-js/lib/compress.js(5761,32): error TS2554: Expected 2 arguments, but got 1.
```

Comment:
this is clearly intended to be optional; the parameter name is `false_by_default`